src/wellcome_aws_utils/reporting_utils.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+get records from VHS, apply the transformation to them, and shove them into
+an elasticsearch index
+"""
+import os
+import json
+import boto3
+import certifi
+from attr import attrs, attrib
+from elasticsearch import Elasticsearch
+from wellcome_aws_utils.lambda_utils import log_on_error
+
+
+def dict_to_location(d):
+    return ObjectLocation(**d)
+
+
+@attrs
+class ObjectLocation(object):
+    namespace = attrib()
+    key = attrib()
+
+
+@attrs
+class HybridRecord(object):
+    id = attrib()
+    version = attrib()
+    location = attrib(converter=dict_to_location)
+
+
+@attrs
+class ElasticsearchRecord(object):
+    id = attrib()
+    doc = attrib()
+
+
+def extract_sns_messages_from_event(event):
```

Comment:
FYI, you also have `extract_sns_messages_from_lambda_event` in `sns_utils.py`, although I’m cringing at it right now.